dotCMS/src/main/webapp/html/js/dotcms/dojo/data/TemplateReadStore.js

```diff
@@ -163,9 +159,16 @@ dojo.declare("dotcms.dojo.data.TemplateReadStore", null, {
 
 	fetchItemByIdentity: function (request) {
 		const templateId = request.identity;
-		const responseEntity = fetch("/api/v1/templates/" + templateId + "/working");
-		const template = responseEntity.entity;
-		this.fetchItemByIdentityCallback(request, template);
+		let response =
+			fetch("/api/v1/templates/" + templateId + "/working")
+				.then(async () => {
```

Comment:
Maybe we should take the response from the arguments of this function

`.then(async ( response ) =>{...`